docs/api/api.md

```diff
@@ -130,23 +137,9 @@ The tokens are returned in alphabetical order by their symbol, so basically, the
 ]
 ```
 
-### `zks_isTokenLiquid`
-
-Given a token address, returns whether it can be used to pay fees.
-
-### Input parameters
-
-| Parameter | Type      | Description               |
-| --------- | --------- | ------------------------- |
-| address   | `address` | The address of the token. |
-
-### Output format
-
-`true`
-
 ### `zks_getL2ToL1MsgProof`
 
-Given a block, a sender and a message, returns the proof for the message sent via the L1Messenger system contract.
+Given a block, a sender, message and optionally log index in block of the L1->L2 message, returns the proof for the message sent via the L1Messenger system contract.
```

Comment:
```suggestion
Given a block, a sender, a message and an optional log index in block of the L1->L2 message, returns the proof for the message sent via the L1Messenger system contract.

Reply:
> log index in block of the L1->L2 message

I don't really get what this means. Probably we want to say **the** block, but some context seems to be missing

Reply:
Yes, the log index in the block. Basically means what is the number of this log in the block